tests/test_sklearn_model_selection.py

```diff
@@ -0,0 +1,61 @@
+"""
+Tests sklearn model selection ops (GridSearchCV, RandomizedSearchCV) converters.
+"""
+import unittest
+import warnings
+
+import numpy as np
+import torch
+from sklearn.tree import DecisionTreeClassifier
+from sklearn.model_selection import GridSearchCV, RandomizedSearchCV
+from sklearn.metrics import make_scorer, accuracy_score
+from sklearn import datasets
+
+import hummingbird.ml
+
+
+class TestSklearnModelSelection(unittest.TestCase):
+    def _test_model_selection(
+        self,
+        model_selection_class,
+        base_classifier,
+        parameters,
+        score_w_train_data=False,
+        **kwargs
+    ):
+        for data in [datasets.load_breast_cancer(), datasets.load_iris()]:
+            X, y = data.data, data.target
+            X = X.astype(np.float32)
+
+            model = model_selection_class(base_classifier, parameters, cv=5, **kwargs)
+
+            n_train_rows = int(X.shape[0] * 0.6)
+            model.fit(X[:n_train_rows, :], y[:n_train_rows])
+
+            if not score_w_train_data:
+                X = X[n_train_rows:, :]
+
+            extra_config = {hummingbird.ml.operator_converters.constants.BATCH_SIZE: X.shape[0]}
```

Comment:
Why is this needed?

Reply:
Agree. It is not needed. So removed it in the latest commit.